README.md

```diff
@@ -26,6 +26,8 @@ Deployment
 ## Run tests 
 - `npm test`
 
+## [Click here for live version](https://summyalena.github.io/Leaderboard/)
+
```

Comment:
[OPTIONAL] 
- [ ] Kindly remember to add the API you consumed for this app in your README.